### PR TITLE
Feature/install report

### PR DESCRIPTION
**Pull request by Lance-Simeon**

### CODE REVIEW

File: BackfillPowerBI.yml [hunk 1]
```diff
@@ -1,14 +1,26 @@
 parameters:
```
Comment (jeff-simeon) [inline BackfillPowerBI.yml:1]: should we have this link to AzurePipelineTemplates like SummaryEmail does? could go either way but worth a discussion

File: BackfillPowerBI.yml [hunk 2]
```diff
@@ -46,3 +74,11 @@ jobs:
       }
     name: Backfill
     displayName: Backfill PowerBI
+
+  - task: PublishPipelineArtifact@1
```
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: purpose? whart artifacts would there be?

File: BackfillPowerBI.yml [hunk 3]
```diff
@@ -1,14 +1,26 @@
 parameters:
+- name: UpdatePowerBIDatasetTableSchema
+  displayName: Update the Power BI Table schema to the latest schema
+  default: true
+  type: boolean
+
 - name: DeleteAllOrganizationData
   displayName: Delete all data in Power BI Simeon Data source
```
Comment (jeff-simeon) [inline BackfillPowerBI.yml]: Simeon Power BI dataset

File: BackfillPowerBI.yml (hunk 2 above)
Comment (Lance-Simeon) [reply BackfillPowerBI.yml]: We are capturing the report before we delete it.

File: BackfillPowerBI.yml (hunk 1 above)
Comment (Lance-Simeon) [reply BackfillPowerBI.yml:1]: moved

**Pull request merged**